### PR TITLE
[IT-655] initial setup of repo for imagecentral account

### PR DESCRIPTION
**Pull request by zaro0508**

### CODE REVIEW

**Comment by zaro0508** [comment]
@ahayden i don't understand your questions.  which references specifically?  which secret specifically?  it would help if you can place your comments next to the code you are referring to.

**Comment by ahayden** [comment]
lol that would be helpful... I was using the mobile client to comment on lines, and it seems it just add them to the end

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+repos:
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v2.3.0
+    hooks:
+    -   id: end-of-file-fixer
+    -   id: trailing-whitespace
+-   repo: https://github.com/adrienverge/yamllint
```
Comment (ahayden) [inline .pre-commit-config.yaml]: references like this

File: config/prod/essentials.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+template_path: "remote/essentials.yaml"
+stack_name: "essentials"
+dependencies:
+  - "prod/bootstrap.yaml"
+parameters:
+  OperatorEmail: "aws.imagecentral@sagebase.org"
+  # VpcPeeringRequesterAwsAccountId: !ssm "/infra/AdmincentralAwsAccountId"
+  VpcPeeringRequesterAwsAccountId: "745159704268"
```
Comment (ahayden) [inline config/prod/essentials.yaml]: Not a secret

File: .pre-commit-config.yaml (hunk 1 above)
Comment (zaro0508) [reply .pre-commit-config.yaml]: i dunno..  i trust git as much as i trust pypi/npm or any other distribution repo.  I mean what's the difference between this and pip install/brew installing any other random open source app?  we are all pretty much hosed aren't we?

File: config/prod/essentials.yaml (hunk 2 above)
Comment (zaro0508) [reply config/prod/essentials.yaml]: AWS does not consider account Ids to be a secret.   There are basically everywhere and you really only need minimal read privileges on an account to access them.

Comment (ahayden) [reply config/prod/essentials.yaml]: agreed- I saw that you commented out reading it from a secret store, so wanted to make sure that's what you intended

**Pull request merged**